src/index.js

```diff
@@ -83,7 +96,10 @@ module.exports = options => ({
     const taggedTemplateLocs = taggedTemplateLocsMap[filepath] || []
     const interpolationLines = interpolationLinesMap[filepath] || []
     const lineCorrection = sourceMapsCorrections[filepath]
-    const warnings = stylelintResult.warnings
+
+    let warnings = stylelintResult.warnings
+
+    warnings = stylelintResult.warnings
```

Comment:
Could you recover these unnecessary changes?

Reply:
Done